Maya/Exporter/GlobalVertex.cs

```diff
@@ -52,37 +52,37 @@ public override bool Equals(object obj)
                 return false;
             }
 
-            if (!other.Position.IsAlmostEqualTo(Position, Tools.Epsilon))
+            if ((Position != null || other.Position != null) && !other.Position.IsAlmostEqualTo(Position, Tools.Epsilon))
```

Comment:
Hey, thanks for the contribution,

I'm not sure if this is logically equivalent to what we talked about.

I was looking for `if ((A.Position == null XOR B.Position ==null) || a.Position !~= B.position)  return false;`

So that if one has the attribute, and the other doesn't, then we can't consider the vertices equivalent.

This change seems to be more like ( if (A or B is not null) && A!=B) return false

Which could work if we did the null checking in `Tools.IsAlmostEqualTo`

But the way that function is implemented, it will cause a runtime exception if we pass it a null value:

https://github.com/BabylonJS/Exporters/blob/356add4f425ce0119f5dbaa1bd5a7a3952993096/Maya/Tools/Tools.cs#L88-L104

Reply:
Makes sense..I reverted it :)

Reply:
oh man yeah sorry you're right! braindead!

Reply:
This is what I had in my head.

```c#
if (!(Position != null && other.Position != null) && ...)
```

Assuming its like C++ and doesn't evaluate the second part

Reply:
It is the case :) Do you mind sending a new PR?